README.md

```diff
@@ -1,4 +1,4 @@
-This action allow you to build your JS file and get only compiled files (run it on node:10:16)
+This action allow you to install and build your JS file and get only compiled files (run it on node:10:16)
```

Comment:
hum not ideal to force nodejs version 10:16 need to find a solution for different version

Reply:
As we're not going to modify the docker tag every month, there are generic tags we could use such as `lts`: https://hub.docker.com/_/node/.